socs/agents/hwp_pmx/agent.py

```diff
@@ -184,7 +208,27 @@ def ign_ext(self, session, params):
         return True, 'Set PMX Kikusui to direct control'
 
     @defer.inlineCallbacks
-    @ocs_agent.param('curr', default=0, type=float, check=lambda x: 0 <= x <= 3)
```

Comment:
Just checking, this is unnecessary (on `set_i`) because the current limit is set? Still seems reasonable to me to have the check in the event the current limit is set wrong.

Reply:
I would like to remove these because it turned out we would like to set larger voltage limit (37V) than the maximum value of 'check' (35V), to avoid the false triggering of the protection.
Also because this device doesn't allow to set the current or voltage larger than its maximum value anyway.